assembl/views/api2/discussion.py

```diff
@@ -675,13 +675,49 @@ def get_time_series_analytics(request):
     return csv_response([r._asdict() for r in results], format, fieldnames)
 
 
+@view_config(context=InstanceContext, name="extract_csv_taxonomy",
+             ctx_instance_class=Discussion, request_method='GET',
+             permission=P_DISC_STATS)
+def extract_taxonomy_csv(request):
+    extracts = db.query(models.Extract).all()
+    extract_list = []
+    filednames = ["Thematic", "Message", "Content Harvested", "Qualify by nature", "Qualify by action",
```

Comment:
There is a typo in your variable name `filednames` :arrow_right: `fieldnames`
Also, don't we want to return column titles in other languages sometimes? (I saw that for phase1 export below we always return French for column titles).

Reply:
well according to the description in the US, multi language support is not necessary